frontend/sumo-championship/src/components/molecules/SeasonItem.tsx

```diff
@@ -5,11 +5,12 @@ type props = {
   name: string;
   startDate: string;
   endDate: string;
+  onClick?: () => void;
```

Comment:
why optional? Isn't it passed everytime?

Reply:
For now yes, but there is no need passing it in case we will reuse this component